airbyte-config/config-models/src/main/resources/types/StandardDestinationDefinition.yaml

```diff
@@ -64,4 +64,5 @@ properties:
     type: string
   supportsDbt:
     type: boolean
-    default: false
+  integrationType:
+    type: string
```

Comment:
This should probably have something that relates to normalization or dbt, e.g. `dbtIntegrationType` or `normalizationIntegrationType`

I'm also noticing that none of the new fields (normalizationRepository, normalizationTag, supportsDbt) have descriptions in this file - we should add them

Reply:
Thank you for the relevant remark. I renamed the field to `normalizationIntegrationType` and added descriptions for all other fields.